cost-analyzer/templates/aggregator-statefulset.yaml

```diff
@@ -44,6 +44,8 @@ spec:
       labels:
         app.kubernetes.io/name: aggregator
         app.kubernetes.io/instance: {{ .Release.Name }}
+        {{/* Force pod restarts on upgrades to ensure the nginx config is current */}}
```

Comment:
```suggestion
        {{/* Force pod restarts on upgrades to ensure configmaps are in sync with cost-model */}}
```

Reply:
What I get for blindly copy and pasting! 🤦